src/atoms/Trackable/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { dataAttributes } from './utils';
+
+function wrappedChild(children, data) {
+  const child = React.Children.only(children);
+
+  if (child.type.displayName === 'Trackable') return child;
```

Comment:
Do you have to use `displayName`? I thought you could just reference the type of the function:
`child.type = Trackable`